OpenEphys.Onix/OpenEphys.Onix/NeuropixelsV2Definitions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace OpenEphys.Onix
+{
+    public enum NeuropixelsV2Probe
+    {
+        ProbeA = 0,
+        ProbeB = 1
+    }
+
+    [Flags]
+    enum NeuropixelsV2Status : uint
+    {
+        SR_OK = 1 << 7 // Indicates the SR chain comparison is OK
+    }
+
+    public static class NeuropixelsV2Definitions
```

Comment:
```suggestion
    static class NeuropixelsV2
```

For consistency with the other device static classes, I would prefer to remove the `Definitions` suffix and internalize this class definition.